src/Data/User/UserDataClass.ts

```diff
@@ -1,8 +1,25 @@
 import { provideLocalStorageDataClass } from '../../utils/provideLocalStorageDataClass'
 import richterImage from './FakeBinaries/richter.jpg'
 import braschauImage from './FakeBinaries/braschau.jpg'
+import { CurrentUser } from '../CurrentUser/CurrentUserDataItem'
```

Comment:
* [ ] FYI depending on the Scrivito data item is reusing one abstraction layer too many, which makes the code harder to understand and less isolated (the dependency on `Scrivito.load` was the yellow flag for me).

Reply:
What do you suggest? Extract the `get` method of `CurrentUserDataItem` and call that method directly?

Reply:
Yes, that sounds good.

Reply:
> Extract the `get` method of `CurrentUserDataItem` and call that method directly?

If we do that we would have to fetch the current user information on every single user call - for get but also for index. If we use `CurrentUserDataItem` instead we benefit from Scrivitos caching.

Reply:
> If we use CurrentUserDataItem instead we benefit from Scrivitos caching.

I see. So we trade a drawback (depending on Scrivito.load) for an advantage (profit from Scrivito caching). Worth a comment?

Reply:
How about:

```
// Use the data item cached by Scrivito instead of connecting to the backend directly
```